src/main/java/org/tikv/common/policy/RetryPolicy.java

```diff
@@ -76,9 +86,11 @@ public RespT callWithRetry(Callable<RespT> proc, String methodName) {
           // add single request duration histogram
           Histogram.Timer requestTimer =
               GRPC_SINGLE_REQUEST_LATENCY.labels(methodName).startTimer();
+          SlowLogSpan slowLogSpan = backOffer.slowLogStart("gRPC " + methodName);
```

Comment:
the method name might be too long to record, would information lose without logging this name?

Reply:
the method name is very usefully for debugging.